### PR TITLE
Platops 2133

### PR DESCRIPTION
**Pull request by DarioGii**

### CODE REVIEW

File: src/main/scala/uk/gov/hmrc/http/HttpDelete.scala [hunk 1]
```diff
@@ -30,9 +30,9 @@ trait HttpDelete
     with HttpHooks
     with Retries {
 
-  override def DELETE[O](url: String)(implicit rds: HttpReads[O], hc: HeaderCarrier, ec: ExecutionContext): Future[O] =
+  override def DELETE[O](url: String, headers: Seq[(String, String)])(implicit rds: HttpReads[O], hc: HeaderCarrier, ec: ExecutionContext): Future[O] =
```
Comment (konradwudkowski) [inline src/main/scala/uk/gov/hmrc/http/HttpDelete.scala]: Would it be worth to keep the API backwards compatible?

Comment (DarioGii) [reply src/main/scala/uk/gov/hmrc/http/HttpDelete.scala]: Do you mean like adding a default parameter?

Comment (konradwudkowski) [reply src/main/scala/uk/gov/hmrc/http/HttpDelete.scala]: yes, that could be a good way to keep it backwards compatible

**Comment by jennygrahamjones** [comment]
Does this supersede #73 ?

**Comment by DarioGii** [comment]
@jennygrahamjones yeah I pulled the changes from that branch and worked from there

**Pull request merged**